### PR TITLE
Standard Tests: store/display contexts on test failures #2184

### LINKED ISSUE #2184: Standard Tests: store/display contexts on test failures
## Tell us about the problem you're trying to solve
If a standard test fails, I want to know all the inputs that led to it failing, the outputs produced by the connector, and the logs.

### PR DESCRIPTION
**Pull request by keu**
closes #2184 

## What
*Describe what the change is solving*
*It helps to add screenshots if it affects the frontend.*

## How
*Describe the solution*

## Pre-merge Checklist
- [ ] *Run integration tests*
- [ ] *Publish Docker images*

## Recommended reading order
1. `test.java`
1. `component.ts`
1. the rest

### CODE REVIEW

File: airbyte-integrations/bases/standard-test/standard_test/utils/connector_runner.py [hunk 1]
```diff
@@ -99,8 +105,10 @@ def run(self, cmd, config=None, state=None, catalog=None, **kwargs) -> Iterable[
         logs = self._client.containers.run(
             image=self._image, command=cmd, working_dir="/data", volumes=volumes, network="host", stdout=True, stderr=True, **kwargs
         )
-        logging.info("Running docker, folders: %s", volumes)
+        logging.info("Docker run: \n%s\ninput: %s\noutput: %s", cmd, self.input_folder, self.output_folder)
+
+        with open(str(self.output_folder / "raw"), "wb+") as f:
```
Comment (sherifnada) [inline airbyte-integrations/bases/standard-test/standard_test/utils/connector_runner.py:110]: How can we prevent this from filling up disk space over time? maybe output should live in `/tmp` by default?

**Review by sherifnada — APPROVED** [review]

File: airbyte-integrations/bases/standard-test/standard_test/utils/connector_runner.py (hunk 1 above)
Comment (keu) [reply airbyte-integrations/bases/standard-test/standard_test/utils/connector_runner.py:110]: output and input folders are in `/tmp` by default, we take base folder from 
`tmp_path` fixture (builtin pytest fixture)
```python
    """Return a temporary directory path object which is unique to each test
    function invocation, created as a sub directory of the base temporary
    directory.
    """

```

Comment (keu) [reply airbyte-integrations/bases/standard-test/standard_test/utils/connector_runner.py:110]: so the output looks like this
```
INFO     root:connector_runner.py:108 Docker run:
read --config tap_config.json --catalog catalog.json
input: /tmp/pytest-of-root/pytest-141/test_two_sequential_reads_inpu0/run_1/input
output: /tmp/pytest-of-root/pytest-141/test_two_sequential_reads_inpu0/run_1/output
```

Comment (sherifnada) [reply airbyte-integrations/bases/standard-test/standard_test/utils/connector_runner.py:110]: great!

**Pull request merged**